analytics/dev/.env.docker-compose

```diff
@@ -1,13 +1,11 @@
-DB_NAME=django
-DB_USER=postgres
-DB_PASS=postgres
-DB_HOST=localhost
-DB_PORT=5432
-OPENSEARCH_ENDPOINT=http://localhost:9200
+DJANGO_SETTINGS_MODULE=analytics.settings.development
+DJANGO_DATABASE_URL=postgres://postgres:postgres@postgres:5432/django
+DJANGO_CELERY_BROKER_URL=redis://redis:6379/
+OPENSEARCH_ENDPOINT=http://elasticsearch:9200
 OPENSEARCH_USERNAME=elastic
 OPENSEARCH_PASSWORD=elastic
 CELERY_BROKER_URL=redis://localhost:6379/0
```

Comment:
Is there a reason for this being included alongside `DJANGO_CELERY_BROKER_URL`? When I run `docker-compose up`, it picks up this env var and fails to connect to redis, as it's not trying to connect to `localhost` instead of `redis`.

Reply:
Probably a bad merge conflict or just a mistake.